main.go

```diff
@@ -93,6 +93,11 @@ func main() {
 	}
 
 	// Migrate the schema
+	err = migration.Migrate(db)
+	if err != nil {
+		panic(err.Error())
```

Comment:
When a cluster of ALM Cores is running under a load balancer and there are multiple transactions, there will be lots of panics, right?